### DISCUSSION TITLE
DJANGO_RQ Error 61 connecting to localhost:6379. Connection refused.

### DISCUSSION THREAD

**Discussion by celstnblacc** [cvat-ai/cvat#2954] [Q&A] [answered]
Hello i try to run the app in debug mode, when i try to create a task i always get this error about django_rq.
How can i start django_rq ?

vscode debug in server:debug
Here is the command lines i use
npm ci && \
cd cvat-core && npm ci && \
cd ../cvat-ui && npm ci && npm start

Thank you

=========================================================

Request Method: | POST
-- | --
http://localhost:7000/api/v1/tasks/49/data
3.1.7
ConnectionError
Error 61 connecting to localhost:6379. Connection refused.

**Comment by azhavoro** [comment, marked as answer]
Hi,
most likely you haven't installed Redis server, please follow this guide: https://github.com/openvinotoolkit/cvat/blob/develop/CONTRIBUTING.md#development-environment

```console
sudo apt-get update && sudo apt-get --no-install-recommends install -y build-essential curl redis-server python3-dev python3-pip python3-venv python3-tk libldap2-dev libsasl2-dev pkg-config libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libswresample-dev libavfilter-dev
```
